templates/bot/js/notification-function-base/src/internal/initialize.js

```diff
@@ -0,0 +1,35 @@
+const { BotFrameworkAdapter } = require("botbuilder");
+const { ConversationBot } = require("@microsoft/teamsfx");
+
+// See https://aka.ms/about-bot-adapter to learn more about adapters.
+const adapter = new BotFrameworkAdapter({
+  appId: process.env.BOT_ID,
+  appPassword: process.env.BOT_PASSWORD,
```

Comment:
Just a question, how are these environment variables set when local debugging?

Reply:
1. local debug will generate a .env.teamsfx.local after registering bot.
<img width="785" alt="image" src="https://user-images.githubusercontent.com/9698542/160749130-306c89d0-da7f-4f8b-b150-bf681d73ebc1.png">
2. `npm run dev:teamsfx` will use this file.

![image](https://user-images.githubusercontent.com/9698542/160749218-1694ccdb-9216-4a79-b0e8-b773a4268918.png)